src/distributions/mod.rs

```diff
@@ -271,6 +271,17 @@ impl<'a, T, D: Distribution<T>> Distribution<T> for &'a D {
 /// than the guarantee some value (`0.0`) is never generated. That makes an open
 /// interval a nicer choice.
 ///
+/// Consider using `Rng::gen_range` if you really need a half-open interval (as
+/// the ranges use a half-open interval). It has the same performance. Example:
```

Comment:
I think we should be clear here that it has the same performance with fixed (0, 1) parameters; it won't in general. Also did we decide to document `gen_range` / `Range` as being half-open?